x-pack/test_serverless/api_integration/test_suites/search/serverless_search/api_key.ts

```diff
@@ -37,7 +47,7 @@ export default function ({ getService }: FtrProviderContext) {
         let apiKeys: SecurityApiKey[];
         // Delete existing API keys
         try {
-          const apiKeysResult = await es.security.getApiKey({ username: kibanaTestUser.username });
+          const apiKeysResult = await es.security.getApiKey({ username: kibanaTestSuperuserServerless.username });
```

Comment:
Changed to use Kibana super user setup for serverless, since we are testing in serverless env